README.md

```diff
@@ -134,32 +138,73 @@ Then `./manage.py translations push` to Transifex and `./manage.py translations
 
 See more detailed instructions in the [how-to guide](https://dsdmoj.atlassian.net/wiki/spaces/laagetaccess/pages/1005060261/Produce+diagram+of+CLA+merits+decision+tree) on Confluence (log-in required).
 
+## Installation in a virtual environment
```

Comment:
@jogilder @MishNajam What do you think of moving this out to a separate documentation and linking to it from here?